src/components/core/BigInput/BigInput.tsx

```diff
@@ -1,40 +1,52 @@
 import { ChangeEventHandler } from 'react';
 import styled from 'styled-components';
 import noop from 'utils/noop';
+import Spacer from '../Spacer/Spacer';
+import ErrorText from '../Text/ErrorText';
 
 type Props = {
   onChange?: ChangeEventHandler<HTMLInputElement>;
   placeholder: string;
   defaultValue?: string;
   autoFocus?: boolean;
+  errorMessage?: string;
 };
 
 function BigInput({
   onChange = noop,
   placeholder,
   defaultValue,
   autoFocus = false,
+  errorMessage,
 }: Props) {
   return (
-    <StyledBigInput
-      autoFocus={autoFocus}
-      placeholder={placeholder}
-      defaultValue={defaultValue}
-      onChange={onChange}
-      autoComplete="off"
-      autoCorrect="off"
-      autoCapitalize="off"
-      spellCheck="false"
-    />
+    <>
+      <StyledBigInput
+        autoFocus={autoFocus}
+        placeholder={placeholder}
+        defaultValue={defaultValue}
+        onChange={onChange}
+        autoComplete="off"
+        autoCorrect="off"
+        autoCapitalize="off"
+        spellCheck="false"
+      />
+      {errorMessage && (
+        <>
+          <Spacer height={4} />
+          <ErrorText message={errorMessage} />
+        </>
+      )}
+    </>
   );
 }
 
 const StyledBigInput = styled.input`
   padding: 0 2px;
   border: 0;
-  font-size: 32px;
-  font-family: 'Times New Roman';
-  line-height: 44px;
+  font-size: 40px;
```

Comment:
I changed these to font-size: 32px and line-height: 44px yesterday based on Jon's feedback in Markup, but figma says 40px and 48px 🤔
https://app.markup.io/markup/9170d884-2ef5-4b68-9b43-203780ceac06

Reply:
gonna ask in discord